db/alembic/versions/b2c5d0a31a8b_add_langgraph_persistence_tables.py

```diff
@@ -0,0 +1,160 @@
+"""add langgraph-persistence tables
+
+Revision ID: b2c5d0a31a8b
+Revises: fa1922057e80
+Create Date: 2025-06-05 12:31:51.290396
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "b2c5d0a31a8b"
+down_revision: Union[str, None] = "fa1922057e80"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
```

Comment:
If I understand correctly, because we’re managing these tables manually, we’ll need to add new migrations whenever LangGraph is upgraded and its checkpoint schema changes. It would be helpful to document this or file a ticket to ensure we don’t overlook it in future updates.